src/Rasterization/Transform/Transform.php

```diff
@@ -202,6 +202,34 @@ public function rotate(float $radians): void
         $this->matrix[3] = $d;
     }
 
+    /**
+     * Calculate the resulting rotation using the yaw rotation matrix
+     *
+     * @return float clockwise rotation
+     */
+    public function rotation(): float
```

Comment:
I think this should be called `getRotation()`, so that there is less potential to confuse it with `rotate()`, and so it starts with a verb (like most functions besides convenience attribute accessors do).